pkg/controller/mizar/mizar-node-controller.go

```diff
@@ -40,12 +40,16 @@ const (
 
 // MizarNodeController points to current controller
 type MizarNodeController struct {
-	// A store of objects, populated by the shared informer passed to MizarNodeController
+	// A store of node objects, populated from Resource partition node informers
 	nodeListers map[string]corelisters.NodeLister
 	// listerSynced returns true if the store has been synced at least once.
 	// Added as a member to the struct to allow injection for testing.
 	nodeListersSynced map[string]cache.InformerSynced
 
+	// A store of node objects, populated from Tenant partition node informer
+	tpNodeListers      corelisters.NodeLister
```

Comment:
nit, this is from single TP, so it should not be plural.

Reply:
a design clarification: does mizar control even care about the tp node?  only nodes under RP are the worker nodes in Arktos so Mizar should only care about the RP nodes.

Reply:
> nit, this is from single TP, so it should not be plural.

When HA is in place, there will be multiple TP master. Even though most of them will be standby, all of them need to have mizar daemon running and registered with mizar.

Reply:
> a design clarification: does mizar control even care about the tp node? only nodes under RP are the worker nodes in Arktos so Mizar should only care about the RP nodes.

Apparently, TP master needs to register as a mizar endpoint. Otherwise, pod allocated to RP won't be able to talk to API server located on TP master.

Reply:
for HA scenario, the multiple master node will be behind a LB, and LB is exposed to the users: in that case, the controller will go to the master node directly? consider a case where a customer pod needs to access the api server, i needs to go to the LB, not the master node directly.

Reply:
Renamed. Leave HA discussion post 130